services/web/src/utils/sentry.js

```diff
@@ -0,0 +1,9 @@
+import * as Sentry from '@sentry/browser';
+import { SENTRY_DSN, ENV_NAME } from 'utils/env';
+
+if (SENTRY_DSN) {
+  Sentry.init({
+    dsn: SENTRY_DSN,
+    enable: !['development', 'test'].includes(ENV_NAME),
```

Comment:
I don't think we need `enable` here as the `SENTRY_DSN` config will be controlling that here... `environment: ENV_NAME` might be needed however.

Reply:
SENTRY_DSN is by default set, regardless of env

Reply:
hmmmmmmmm... it's a bit counter to how we use env configs but I see now it would be the same for all environments so...maybe? either way I guess `environment: ENV_NAME` is still required to differentiate the envrionments.

Reply:
yep e.g. in acrylic we got demo too, i do want to come back to revisit env.js but for later

Reply:
on second thought can we have `SENTRY_DSN` control the loading instead? This way is so much simpler. That way we can have it enabled for non-standard environments like `demo` and also enable it for `development` if we want to

Reply:
I am not sure i follow what do you mean by loading? (is that the enable flag)

Reply:
meaning we leave `SENTRY_DSN` null for dev and only initialize when the config is set

Reply:
I am not a fan of that because that means you have to duplicate the SENTRY_DSN for each environment, because the default is null

Reply:
right but that's generally how our environment configs work no? it's expected behavior. Also having the default be null makes sense as sentry is off in development

Reply:
I guess I wouldn't mind this too:

```
if (SENTRY_DSN && ENV_NAME !== 'development') {
  Sentry.init(...)
}
```

but basing functionality on `ENV_NAME` is a bit of a smell too as we discussed before...
this is why loading it by CDN was a lot easier :) ... I'm not sure the difference either since either way it's locked to a version

Reply:
sure we can do that, the enable flag would do the same was my thinking